modules/cnv-vm-storage-volume-types.adoc

```diff
@@ -5,9 +5,10 @@
 [id="cnv-vm-storage-volume-types_{context}"]
 = Virtual machine storage volume types
 
-Virtual machine storage volume types are listed, as well as domain and volume settings. See the
-https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachinespec[kubevirt
-API Reference] for a definitive list of virtual machine settings.
+Virtual machine storage volume types are listed, as well as domain and volume settings. +
```

Comment:
would this show only in the storage section, not the virt one?

How about defining a variable for `KUBEVIRT_VERSION_SHIPPED=v0.26.5` (or something like this) so it is easier to remember to update every type we release, and easier to refer in multiple places (now only two)

Reply:
Yeah, this PR is just a quick fix to remove the issue in the current published docs - targetted to enterprise-4.4 only. The link is in the storage types module as part of the 'creating a VM' assembly.

For the long term I like the idea of a kubevirt version variable for the docs. Is that something we can rely on going forward? If we don't mind linking to the kubevirt API reference in future versions - just as long as it's linked to the appropriate version - then that is a fine solution. We could add the variable to our pre-flight checklist to ensure it gets updated.

Reply:
I think that there is a lot of value in explicitly stating the version of KubeVirt used - as long as we keep the language defensive. It has to be clear that not everything that is upstream is supported within OpenShift. I'd prefer if the same language is kept on the virt side, too.

Reply:
@dankenigsberg I've moved it out of the storage section and at end of the 'create VM' assembly. There is now only one reference/link to the API reference, and it includes a note calling out that not everything in the API reference is supported.
Using explicit version here because it's published doc. When this is acked/merged I'll do the same in master with the KUBEVIRT_VERSION variable. 

Screenshot of built doc:
![Screenshot from 2020-05-12 12-42-37](https://user-images.githubusercontent.com/17755748/81675926-ad5b7200-944f-11ea-9371-d0921964e065.png)